README.md

```diff
@@ -52,12 +52,13 @@ identified from [commit messages][3]:
 
 ## Inputs
 
-| Key         | Description                                            | Default value         |
-| ----------- | ------------------------------------------------------ | --------------------- |
-| `label`     | Build label for the development iteration              | `.dev`                |
-| `changelog` | Set true to create a changelog-X.md file               | `false`               |
-| `preset`    | Preset for creating the changelog                      | `conventionalcommits` |
-| `folder`    | Folder in the workspace in which the .git repo resides | `./`                  |
+| Key             | Description                                                           | Default value         |
+| --------------- | --------------------------------------------------------------------- | --------------------- |
+| `label`         | Build label for the development iteration                             | `.dev`                |
+| `changelog`     | Set true to create a changelog-X.md file                              | `false`               |
+| `preset`        | Preset for creating the changelog                                     | `conventionalcommits` |
+| `folder`        | Folder in the workspace in which the .git repo resides                | `./`                  |
+| `bumpoverride`  | Override the version bump, can be either `major`, `minor` or `patch`  |                       |
```

Comment:
maybe we should add `auto` too, `auto` is the default for automatic decisions based on tags and commits.